### PR TITLE
PIN-3232 [BE] [Tenant Management] Salvare tipologia ente

### PR DESCRIPTION
**Pull request by nttdata-rtorsoli**

### CODE REVIEW

File: models/src/main/scala/it/pagopa/interop/tenantmanagement/model/persistence/JsonFormats.scala [hunk 1]
```diff
@@ -80,8 +80,23 @@ object JsonFormats {
       }
     }
 
+  private implicit val ptkFormat: RootJsonFormat[PersistentTenantKind] =
+    new RootJsonFormat[PersistentTenantKind] {
+      override def read(json: JsValue): PersistentTenantKind = json match {
+        case JsString("PA")      => PersistentTenantKind.Pa
+        case JsString("GSP")     => PersistentTenantKind.Gsp
```
Comment (beetlecrunch) [inline models/src/main/scala/it/pagopa/interop/tenantmanagement/model/persistence/JsonFormats.scala:87]: May I suggest using upper case for every character in acronyms?

File: models/src/main/scala/it/pagopa/interop/tenantmanagement/model/tenant/PersistentTenantKind.scala [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package it.pagopa.interop.tenantmanagement.model.tenant
+
+sealed trait PersistentTenantKind
+
+object PersistentTenantKind {
+  case object Pa      extends PersistentTenantKind
+  case object Gsp     extends PersistentTenantKind
```
Comment (beetlecrunch) [inline models/src/main/scala/it/pagopa/interop/tenantmanagement/model/tenant/PersistentTenantKind.scala]: May I suggest using upper case for every character in acronyms?

**Review by beetlecrunch — APPROVED** [review]

File: src/main/resources/interface-specification.yml [hunk 3]
```diff
@@ -611,6 +619,8 @@ components:
             $ref: '#/components/schemas/TenantAttribute'
         name:
           type: string
+        kind:
```
Comment (galales) [inline src/main/resources/interface-specification.yml:622]: `kind` must be mandatory in the seed

File: src/main/resources/interface-specification.yml [hunk 4]
```diff
@@ -630,6 +640,8 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/MailSeed'
+        kind:
```
Comment (galales) [inline src/main/resources/interface-specification.yml:644]: same, `kind` must be mandatory in the seed

**Review by galales — CHANGES_REQUESTED** [review]

**Review by TonioGela — APPROVED** [review]

**Review by galales — APPROVED** [review]

**Pull request merged**